lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -120,7 +121,20 @@ def check_alt_backend(alt_backend):
 fig.canvas.mpl_connect("draw_event", lambda event: timer.start())
 fig.canvas.mpl_connect("close_event", print)
 
+result = io.BytesIO()
+fig.savefig(result, format='png')
+
 plt.show()
+
+# Ensure that the window is really closed.
+plt.pause(0.5)
```

Comment:
BTW, if anyone can think of a better way to do this, I'd change it.

I tried `fig.canvas.flush_events()`, but that crashes because the main window is gone. If you ignore any crashes, then the test doesn't fail like it should on `master`.